plugins/modules/ec2_group.py

```diff
@@ -105,6 +110,20 @@
             - The end of the range of ports that traffic is coming from.
             - A value can be between C(0) to C(65535).
             - A value of C(-1) indicates all ports (only supported when I(proto=icmp)).
+        icmp_type:
+          version_added: 3.3.0
+          type: int
+          description:
+          - When using C(proto: icmp) or C(proto: icmpv6), allows you to
```

Comment:
```
2022-04-22 22:56:43.230449 | controller | ERROR: Found 2 yamllint issue(s) which need to be resolved:
2022-04-22 22:56:43.230573 | controller | ERROR: plugins/modules/ec2_group.py:117:49: error: DOCUMENTATION: syntax error: mapping values are not allowed here (syntax)
2022-04-22 22:56:43.230844 | controller | ERROR: plugins/modules/ec2_group.py:117:49: unparsable-with-libyaml: None - mapping values are not allowed in this context
```

This is likely what triggered the big slew of errors, the `:` is interpreted by YAML as the divider between the key and value.  (this catches me out too)

Reply:
This is a good catch. It seems that treating colons as dividers between keys and values regardless of where they are is entirely up the parser. It never occurred to me that this would happen. I'll use the equal sign.